website/settings/defaults.py

```diff
@@ -158,7 +158,8 @@ def parent_dir(path):
 SHORT_DOMAIN = 'osf.io'
 
 # TODO: Combine Python and JavaScript config
-COMMENT_MAXLENGTH = 500
+# Add 500 to account for variable length due to at mentions
+COMMENT_MAXLENGTH = 500 + 500
```

Comment:
I'd like to think about this. To start with, it no longer matches `MAXLENGTH` [in JavaScript](https://github.com/CenterForOpenScience/osf.io/blob/eac466dba48543c997dbcbecc4ed2955e41ae23e/website/static/js/comment.js#L200).